src/Microsoft.Xaml.Behaviors.DesignTools/Microsoft.Xaml.Behaviors.DesignTools.csproj

```diff
@@ -63,6 +62,7 @@
       <Generator>ResXFileCodeGenerator</Generator>
       <LastGenOutput>Resources.Designer.cs</LastGenOutput>
       <SubType>Designer</SubType>
+      <CustomToolNamespace>Microsoft.Xaml.Behaviors.DesignTools</CustomToolNamespace>
     </EmbeddedResource>
```

Comment:
Any particular reason to change this?

Reply:
It was made the same across the two projects.
Having it at the root eliminates the need for a using statement.

Reply:
And added the need for a custom tool namespace...  ;-)

Anyway, it's fine I was just curious.

---
In reply to: [349328568](https://github.com/microsoft/XamlBehaviorsWpf/pull/60#discussion_r349328568) [](ancestors = 349328568)